velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -330,20 +329,38 @@ bool containsTypeKind(const TypePtr& type, const TypeKind& search) {
   return false;
 }
 
+bool containsType(const TypePtr& type, const TypePtr& search) {
+  if (type == search) {
```

Comment:
Should we use .equivalent here?

Reply:
I can change it to .equivalent here, but I don't think using == would be a problem. I found that Type::operator== calls this->equivalent(). The only types that override operator== are RowType and OpaqueType. For RowType, since it's not enumerable, we should use the containsTypeKind() API anyway. For OpaqueType, operator== checks that both typeIndex_ match in addition to calling this->equivalent(). Plus, we don't verify results for Opaque type using DuckDB either.